src/main/java/edu/ie3/datamodel/io/source/csv/CsvDataSource.java

```diff
@@ -266,7 +275,7 @@ private OperatorInput getFirstOrDefaultOperator(
    *
    * }</pre>
    *
-   * ...
+   * <p>...
```

Comment:
This is no proper syntax. Even the XHTML-shorthand `<p/>` would be discouraged.

```suggestion
   * <p>...</p>
```